src/sections/About/About.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import './About.css';
+import UserProfile from '../../assets/images/UserProfile.png'
+import AppStoreButtons from '../../assets/AppStoreButtons/AppStoreButtons';
+
+const About = () => {
+    return (
+        <>
+            <section>
+                <div className='about-text'>
+                    <h1>Connecting those in Need</h1>
+                    <p>When one of us has food insecurity, it is a problem for all of us. Let’s change that, one meal at a time. <br /><br />
```

Comment:
Remove break from this paragraph please :)

Reply:
HI, have done :D